.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
in our team, we had a session at the beginning of the project to agree on a formatting template that we were all happy with for the project instead of having it enforced. Having a company wide set of formatting rules may be heavy handed, the best we could aim for may be a consistent set of tools to use, in which the settings can be chosen by individual teams.